test/Riok.Mapperly.IntegrationTests/Mapper/TestMapper.cs

```diff
@@ -41,8 +41,6 @@ public TestMapper()
 
         public partial long ImplicitCastInt(int value);
 
-        public partial int ExplicitCastInt(uint value);
```

Comment:
Here, the explicit cast was tested explicitly. Instead of removing it, I'd enable the explicit cast (using `.All`) for this mapper.